cmd/oras/root/manifest/fetch.go

```diff
@@ -58,6 +58,9 @@ Example - Fetch the descriptor of a manifest from a registry:
 Example - Fetch the manifest digest from a registry similar to the resolve command:
   oras manifest fetch --format go-template --template '{{ .digest }}' localhost:5000/hello:v1
 
+Example - Fetch manifest and format output with JSON:
```

Comment:
manifest itself is JSON.
```suggestion
Example - Fetch manifest and output metadata encoded in JSON:
```